### DISCUSSION TITLE
My account is flagged, help me

### DISCUSSION THREAD

**Discussion by vinascofran14** [community/community#32356] [New to GitHub] [answered]
my account is flagged and I have no idea, I'm new to this and it already looks like this.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the Community is unable to assist with flagged accounts :)

**Reply by Zibri** [reply]
I did.. but it's 6 hours and I got no reply... I don't even know if it happened because of some glitcgh or some limit I hit or something bad I did (which is unlikely).

**Reply by zihanghliu** [reply]
I was also flagged for no reason and got no reply. are there any other ways to resolve this?

**Reply by tomfrancix** [reply]
I too was flagged and no reason was given.

**Reply by Abhishekvk04** [reply]
Same here my account is also flagged and no one can see my profile. while they are getting 404 error. I have raised 4 tickets till now from a month but no response on the other end . pls help me resolve this issue

**Reply by bbkxm** [reply]
I did.. but it's 2 weeks and I got no reply...

**Reply by Nora-rad** [reply]
I got no reply too

**Reply by CHARLESUKE** [reply]
Good afternoon, I noticed my github has been flagged without any prior notifications and for what exactly..This has caused me some reduction in my project scores and which is affecting my total score. I am really not happy about this and want this to be resolved immediately. why is github making things so difficult!!!

**Reply by sahaj-96** [reply]
@CHARLESUKE how much time did it take for them to unflag ur account??? facing a similar issue right now?

**Reply by sanjeevmansotra1** [reply]
The time it takes for GitHub to unflag an account can vary. It typically depends on the specific circumstances and the reason for the flag. Generally, the process might take anywhere from a few hours to a few days. If you need to expedite the process, contacting GitHub Support directly and providing all necessary information can help. They usually respond to support tickets within 24-48 hours.

**Reply by fgsfgh** [reply]
No, they don't. I created the ticket on May 24. Now it's the other half of August and still zero reaction from these rascals. Not a single F is given. I can't even post from it here.

I wouldn't recommend GH to anyone. It's an utter trash and zero responsibility. They "don't tolerate the bulling", but they're totally ok with vandalizing repos and spoiling technical discussions. And if you get provoked and flagged, forget about the account. They don't care at all.

**Reply by lespinozaguerrero6-png** [reply]
This no  say developers

**Comment by Marioacm1208** [comment]
Same here! GitHub is flagging accounts for no reason, I've opened a ticket more than a week ago and nobody in support seems to care. I need to get back to work, I'm paying for my organization's GitHub Copilot and I'm losing my benefit since now I cannot even accept invitations to organizations. (Yes dumb me exited from organization as part of troubleshooting steps).

**Reply by Marioacm1208** [reply]
![image](https://github.com/community/community/assets/17240923/4abc664e-39ad-4b87-b592-ab6e337dd8cf)


8 days later to be exactly and no solution has been provided so this is what you can expect body

**Comment by phamdung2209** [comment]
My account is also flagged, what should I do?

**Comment by SudhiSumesh** [comment]
my account is flagged ,it can't shown by plublic ,how can I solve these

**Reply by SanDevil23** [reply]
is it resolved now ?? i am facing the same problem since a week.

**Reply by GabrielADKN** [reply]
Same things, I have opened a ticket more than a week ago but nobody has responded.

**Reply by shihabmuhtasim** [reply]
Is it resolved? I'm facing the issue for over 2 weeks. No response from support team

**Reply by GabrielADKN** [reply]
> Same things, I have opened a ticket more than a week ago but nobody has responded.

Still no answer from the support team after 2 weeks now

**Reply by FigmentBoy** [reply]
Same here, I'm yet to receive a response

**Reply by FigmentBoy** [reply]
Although I am able to see your profile if that helps

**Reply by SA-j00u** [reply]
2 weeks...

**Reply by GAP-Pathum** [reply]
How to know the reason for flagging the account ?

**Reply by G0r3s170** [reply]
It is said that you get flagged because of suspicious activity (the bots do that) or creation of multiple repositories on the same net

**Comment by Abhishekvk04** [comment]
My account is also flagged. Pls help me to resolve this.

**Comment by rovsanhashimov** [comment]
Hello. I want to unlink my Github account from the Galxe website. Help me do this

**Comment by VirtualCard** [comment]
My account is also flagged. Pls help me to resolve this.

**Comment by poisonflood** [comment]
my account is flagged and github dev reply "
1. add a non-disposable email address on the account
2. remove any throwaway emails (including forwarders
"
I have add a gmail and remove  temp email.
then My account is stll banned!!!

**Comment by Sharon20001212** [comment]
How to solve his problem?

**Comment by abobsasasasa18888** [comment]
yo

**Comment by suleimanq** [comment]
my account also flagged.

**Reply by suleimanq** [reply]
how to solve this problem?

**Comment by aakvatech** [comment]
My account is also flagged Please assist

**Comment by Imran-imtiaz48** [comment]
my account is flagged ,it can't shown by public ,how can I solve these

**Comment by zaryabjiya** [comment]
My account has been flagged, and I am not sure why. Could you please provide me with more details or guide me on how to resolve this issue?

**Comment by KSue-Dev** [comment]
Yeah, my account WAS flagged even I'm just new here 3 weeks ago.

**Reply by SA-j00u** [reply]
hmm but you account is visible
or you unblocked only in less than 19 hours?
or this post only published after you unblocked?

**Comment by ymbu-technology** [comment]
My account was also flagged. I don't understand why. It was a brand-new account. Now I can create repositories but I cannot invite anyone to contribute to the project. I also cannot delete the account because of the flag. 

Any answer? Does anyone in the same situation get an answer from GitHub?

**Reply by Dinogecko** [reply]
im have the same issue they only sent an email 4 days later saying "suspicious activity" and called it closed
